doc/cephadm/certmgr.rst

```diff
@@ -103,54 +111,54 @@ Cephadm certmgr supports three different scopes for certificate management:
 
 1. **Global Scope:**
    - Certificates in this scope are shared across all service daemons, regardless of which host they are running on.
-   - Example: `mgmt-gateway` certificate is a globally shared certificate used by all service daemons.
+   - Example: ``mgmt-gateway`` certificate is a globally shared certificate used by all service daemons.
 
 2. **Per-Host Scope:**
    - Certificates are assigned per host, meaning each host has its own unique certificate.
    - When configuring a custom certificate, the user must specify the host for which the certificate applies.
-   - Example: `grafana` service cerificates are configured at the host level and applies specifically to a single machine.
+   - Example: ``grafana`` service certificates are configured at the host level and apply specifically to a single machine.
 
 3. **Per-Service Scope:**
-   - Certificates are configured per service name, meaning each instance of a service can have its own certificate.
+   - Certificates are configured per service name (each service instance can have its own certificate).
    - When specifying a custom certificate, the user must define the service to which it belongs.
-   - Example: an `rgw` service certificate is assigned specifically and only to an RGW service
+   - Example: an ``rgw`` service certificate is assigned specifically and only to an RGW service
 
 
-Reloading Certificate Manager
-=============================
+Listing Certificates
+====================
 
-Run the following command to reload the certificate manager configuration from the Monitors:
+To list all certificates managed by Cephadm:
 
 .. prompt:: bash #
 
-   ceph orch certmgr reload
-
-This command ensures that any changes made to certificate configurations are
-applied immediately without requiring a service restart.
+   ceph orch certmgr cert ls [--show-details] [--include-cephadm-signed] [--filter-by <expression>]
 
-Listing Certificates
-====================
+This command displays an overview of all certificates currently managed by
+Cephadm. The ``--show-details`` option includes additional information such as
+issuing authorities and certificate extensions.
 
-To list all available certificates managed by the orchestrator:
+By default, this command omits Cephadm-signed certificates. To include them,
+pass the ``--include-cephadm-signed`` argument.
 
-.. prompt:: bash #
+Examples:
 
-  ceph orch certmgr cert ls [--show-details]
+.. code-block:: bash
```

Comment:
Perhaps use prompt block here like the other commands? I see it used to be that before but was changed -- maybe I'm missing some nuance in this case.

```suggestion
.. prompt:: bash #
```

Reply:
Fixed